projects/media-viewer/src/lib/annotations/comment-set/comment-set-render.service.ts

```diff
@@ -50,9 +54,25 @@ export class CommentSetRenderService {
     }
   }
 
-  private height(element:any) {
+  private height(element: any) {
     return element.form.nativeElement.getBoundingClientRect().height;
   }
 
   private difference(a: number, b: number): number { return Math.abs(a - b); }
+
+  private makeSureWithinContainer(commentItem: CommentComponent, previousCommentItem: CommentComponent, containerHeight: number) {
+    containerHeight -= 10;
+    if (commentItem.commentBottomPos > containerHeight) {
+      commentItem.rectTop -= commentItem.commentBottomPos - containerHeight;
+    }
+    if (previousCommentItem) {
```

Comment:
I feel like this should be an if-else because the first if is only relevant the first-time when prevComment is null. all subsequent times, prevComment has a value and bottom comment has already been reset, so no need for it to be checked again

Reply:
I'll make the change